cmd/root.go

```diff
@@ -0,0 +1,233 @@
+// Copyright © 2017 The virtual-kubelet authors
```

Comment:
do we really need all this? We want a binary with some flags that implements the Provider interface? That would just be some `cobra` stuff

Reply:
Implementing the Provider interface is not enough, and that's why node-cli came to be. So yes, this code is very much needed and until node-cli is reworked as part of VK into pieces you can pick, I can't see other option other than having this here.

Reply:
Miek's gave a nod to my last comment https://github.com/virtual-kubelet/systemk/pull/65#issuecomment-763473080